src/diy/official_clone_trackers.md

```diff
@@ -0,0 +1,113 @@
+From here you can do a one click order.
+You should be able to leave all the settings as is aside from Surface Finish, set this to LeadFree HASL or ENIG!
+*You can also change the color of the solder mask if you so desire.*
+After this you should be able to continue with the ordering process and then the wait begins!
+
+**BEWARE**
+Sometimes when you get redirected to JLCPCB it won't fully load, in this case you might have to try again.
+
+## Soldering the IMU to the main PCB
+The soldering process for official slimes is a little different than your usual DIY slimes. Where normally DIY slimes are soldered together with through hole wiring, SlimeVR on the other hand uses castelated edges. This makes soldering the IMU very fast and efficient.
+
+What do you need to solder the IMU to the main PCB?
+- Soldering iron.
+- Leadfree Solder.
+- Chisel style soldering tip (optional but recommended).
+- Assembly jig (optional).
+
+Instructions:
+Place the IMU module on top of the main tracker PCB as shown below.
+![image](../assets/img/tracker_IMU_orientation.png)
+Now solder the 6 castelated edges to the main board as shown in the picture above.
+Using a chisel tip makes this process significantly faster and easier.
+For more soldering tips you can check out this video:
+<div class="video-container">
+<iframe width="100%" height="auto" src="https://www.youtube.com/embed/P0YX_eKyfxA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay muted; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+
+## Flashing the firmware
+To flash the firmware you will need:
+- USB-C cable with data pins (official SlimeVR cable is recommended).
+- Paperclip or tweezers to bridge a connection on the PCB.
+
+Flashing the firmware for these official trackers is practically identical to the process of [flashing DIY trackers](https://docs.slimevr.dev/firmware/index.html).
+A link to the online flasher with all the settings pre-configured can be found [here](https://slimevr-firmware.bscotch.ca/?config=eyJib2FyZCI6eyJ0eXBlIjoiQk9BUkRfU0xJTUVWUiIsInBpbnMiOnsiaW11U0RBIjoiMTQiLCJpbXVTQ0wiOiIxMiIsImxlZCI6IjIifSwiZW5hYmxlTGVkIjp0cnVlfSwiaW11cyI6W3sidHlwZSI6IklNVV9CTk8wODUiLCJpbXVJTlQiOiIxNiIsImVuYWJsZWQiOnRydWUsInJvdGF0aW9uIjoiMjcwIn0seyJlbmFibGVkIjp0cnVlLCJ0eXBlIjoiSU1VX0JOTzA4NSIsInJvdGF0aW9uIjoiMjcwIiwiaW11SU5UIjoiMTMifV0sImJhdHRlcnkiOnsidHlwZSI6IkJBVF9FWFRFUk5BTCIsInJlc2lzdGFuY2UiOjE4MCwicGluIjoiMTcifSwidmVyc2lvbiI6IlNsaW1lVlIvbWFpbiJ9).
+If the link does not work or the settings don't show up right after clicking you can find the manual settings a little further down in this section.
+As opposed to DIY trackers the official trackers need to be started in flash mode. You can do this by bridging the pads on the top side of the PCB as show below:

```

Comment:
Show example of bridging or draw on picture. Maybe add the bridge to shielding of ESP trick.

Reply:
Not sure if that's really necessary as the actual bridging pads have been shown right above this and it's no more than just touching both pads together with tweezers.
However what is this shielding ESP trick you mentioned?

Reply:
Bridging one of the pads (I think it was the one closest to the edge) to the shielding of the ESP rather than the other pad, this is easier and more reliable to do than to the other pad.

Reply:
That is fair, however the pads have worked well for me in the past and I feel like we should probably stick to the official method and guide people if they have issues.